chapter02/jmw/src/main/java/spring/training/services/BankService.java

```diff
@@ -0,0 +1,43 @@
+
+package spring.training.services;
+
+import java.math.BigDecimal;
+
+import spring.training.model.Account;
+import spring.training.model.AccountTransaction;
+
+public class BankService {
+
```

Comment:
I thought, the instruction talks about having an interface. 
Anyway I feel, having a service without interface is a bad design :)